index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+const teams = require('./teams.json')
+
+const app = express()
+
+app.get('/teams', (request, response) => {
+    response.send(teams)
+})
+
+app.get('/teams/:id', (request, response) => {
+    let abbreviationId
+    let teamId
+    abbreviationId = teams.filter((team) => {
+        return team.abbreviation === (request.params.id.toUpperCase())
+    })
+
+    teamId = teams.filter((team) => {
+        return team.id === parseInt(request.params.id)
+
+    })
+    if (!isNaN(request.params.id)) {
+        response.send(teamId)
+    } else {
+        response.send(abbreviationId)
+    }
```

Comment:
Since you are always sending a list, if there were no matching teams, you'd be sending a 200 with an empty list instead of a 404.